include/h3api/H3Constants/H3CstCombat.hpp

```diff
@@ -109,10 +109,102 @@ namespace h3
 
         constexpr INT32 COMBATSQUARE_WIDTH  = 44;
         constexpr INT32 COMBATSQUARE_HEIGHT = 42;
+
+        namespace NMagicAnimation
+        {
+            enum eMagicAnimation : INT32
+            {
+                PRAYER,
+                LIGHTNING_BOLT,
+                AIR_SHIELD,
+                BACKLASH,
+                ANIMATE_DEAD,
+                ANTI_MAGIC,
+                BLIND,
+                COUNTERSTROKE,
+                DEATH_RIPPLE,
+                FIREBLAST,
+                DECAY,
+                FIRE_SHIELD,
+                FIRESTORM,
+                DISRUPTIVERAY_RAY,
+                DISRUPTIVERAY_BURST,
+                FEAR,
+                METEOR_SHOWER,
+                FRENZY,
+                FORTUNE,
+                MUCK_AND_MIRE,
+                MIRTH,
+                HYPNOTIZE,
+                PROTECTION_FROM_AIR,
+                PROTECTION_FROM_WATER,
+                PROTECTION_FROM_FIRE,
+                PRECISION,
+                PROTECTION_FROM_EARTH,
+                SHIELD,
+                SLAYER,
+                SACRED_BREATH,
+                SORROW,
+                TAIL_WIND,
+                FORCEFIELD_2,
+                FORCEFIELD_3,
+                REMOVE_OBSTACLE,
+                BERSERK,
+                BLESS,
+                CHAIN_LIGHTNING_BOLT,
+                CHAIN_LIGHTNING_DUST,
+                CURE,
+                CURSE,
+                DISPEL,
+                FORGETFULNESS,
+                FIREWALL_2,
+                FIREWALL_3,
+                FROSTRING,
+                ICERAY_BURST,
+                LAND_MINE,
+                MISFORTUNE,
+                LIGHTNING_DUST,
+                RESURRECTION,
+                SACRIFICE_SLAY,
+                SACRIFICE_RESURRECT,
+                SPONTANEOUS_COMBUSTION,
+                TOUGH_SKIN,
+                QUICKSAND,
+                WEAKNESS,
+                LAND_MINE_EXPLOSION,
+                DISPEL_QUICKSAND,
+                DISPEL_LAND_MINE,
+                DISPEL_FORCEFIELD_2,
+                DISPEL_FORCEFIELD_3,
+                DISPEL_FIREWALL_2,
+                DISPEL_FIREWALL_3,
+                MAGICBOLT_BURST,
+                FIREWALL_1,
+                DISPELFIREWALL_1,
+                POISON,
+                BIND,
+                DISEASE,
+                PARALYZE,
+                AGE,
+                DEATH_CLOUD,
+                DEATH_BLOW,
+                DRAIN_LIFE,
+                MAGICCHANNEL_SUCK,
+                MAGICCHANNEL_SPEW,
+                MAGIC_DRAIN,
+                MAGIC_RESISTANCE,
+                REGENERATE,
+                DEATH_STARE,
+                ACID_BREATH,
+                POOF,
```

Comment:
Enums usually have their value next to them. I made a script to handle this automatically few years back thankfully.
Hoping spacing is correct this time!
```suggestion
                PRAYER = 0,
                LIGHTNING_BOLT = 1,
                AIR_SHIELD = 2,
                BACKLASH = 3,
                ANIMATE_DEAD = 4,
                ANTI_MAGIC = 5,
                BLIND = 6,
                COUNTERSTROKE = 7,
                DEATH_RIPPLE = 8,
                FIREBLAST = 9,
                DECAY = 10,
                FIRE_SHIELD = 11,
                FIRESTORM = 12,
                DISRUPTIVERAY_RAY = 13,
                DISRUPTIVERAY_BURST = 14,
                FEAR = 15,
                METEOR_SHOWER = 16,
                FRENZY = 17,
                FORTUNE = 18,
                MUCK_AND_MIRE = 19,
                MIRTH = 20,
                HYPNOTIZE = 21,
                PROTECTION_FROM_AIR = 22,
                PROTECTION_FROM_WATER = 23,
                PROTECTION_FROM_FIRE = 24,
                PRECISION = 25,
                PROTECTION_FROM_EARTH = 26,
                SHIELD = 27,
                SLAYER = 28,
                SACRED_BREATH = 29,
                SORROW = 30,
                TAIL_WIND = 31,
                FORCEFIELD_2 = 32,
                FORCEFIELD_3 = 33,
                REMOVE_OBSTACLE = 34,
                BERSERK = 35,
                BLESS = 36,
                CHAIN_LIGHTNING_BOLT = 37,
                CHAIN_LIGHTNING_DUST = 38,
                CURE = 39,
                CURSE = 40,
                DISPEL = 41,
                FORGETFULNESS = 42,
                FIREWALL_2 = 43,
                FIREWALL_3 = 44,
                FROSTRING = 45,
                ICERAY_BURST = 46,
                LAND_MINE = 47,
                MISFORTUNE = 48,
                LIGHTNING_DUST = 49,
                RESURRECTION = 50,
                SACRIFICE_SLAY = 51,
                SACRIFICE_RESURRECT = 52,
                SPONTANEOUS_COMBUSTION = 53,
                TOUGH_SKIN = 54,
                QUICKSAND = 55,
                WEAKNESS = 56,
                LAND_MINE_EXPLOSION = 57,
                DISPEL_QUICKSAND = 58,
                DISPEL_LAND_MINE = 59,
                DISPEL_FORCEFIELD_2 = 60,
                DISPEL_FORCEFIELD_3 = 61,
                DISPEL_FIREWALL_2 = 62,
                DISPEL_FIREWALL_3 = 63,
                MAGICBOLT_BURST = 64,
                FIREWALL_1 = 65,
                DISPELFIREWALL_1 = 66,
                POISON = 67,
                BIND = 68,
                DISEASE = 69,
                PARALYZE = 70,
                AGE = 71,
                DEATH_CLOUD = 72,
                DEATH_BLOW = 73,
                DRAIN_LIFE = 74,
                MAGICCHANNEL_SUCK = 75,
                MAGICCHANNEL_SPEW = 76,
                MAGIC_DRAIN = 77,
                MAGIC_RESISTANCE = 78,
                REGENERATE = 79,
                DEATH_STARE = 80,
                ACID_BREATH = 81,
                POOF = 82,
```